source/app/iris_engine/reporter/ImageHandler.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+#
+#  IRIS Source Code
+#  Copyright (C) 2021 - Airbus CyberSecurity (SAS)
```

Comment:
Since you wrote this file,  you can set the licensing as following 

```suggestion
#  contact@dfir-iris.org
#  Created by Lukas Zurschmiede @LukyLuke
```

Reply:
Do you also want the copyright removed?
Or just the "created by" added as I did.

Reply:
You can also remove the Copyright 👍